docs/EN_US/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml

```diff
@@ -6298,6 +6298,134 @@ ecl zapgen W20171018-091399 --path ~ --inc-thor-slave-logs --description "Unexpe
+        ensuring that the operating system does not swap the memory
+        out.</para>
+
+        <para>Examples:</para>
+
+        <programlisting>ecl roxie memlock mycluster
+</programlisting>
+      </sect2>
+
+      <sect2 id="CT-RCTL-MEMUNLOCK" role="brk">
+        <title>ecl roxie memunlock</title>
+
+        <para><emphasis role="bold">ecl roxie memunlock
+        &lt;process_cluster&gt;</emphasis></para>
+
+        <informaltable colsep="1" frame="all" rowsep="1">
+          <tgroup cols="2">
+            <colspec colwidth="80.50pt" />
+
+            <colspec />
+
+            <tbody>
+              <row>
+                <entry><emphasis>Options</emphasis></entry>
+
+                <entry></entry>
+              </row>
+
+              <row>
+                <entry><emphasis>process_cluster</emphasis></entry>
+
+                <entry>The roxie process cluster to memunlock</entry>
+              </row>
+            </tbody>
+          </tgroup>
+        </informaltable>
+
+        <para>The <emphasis>ecl roxie memunlock</emphasis> command unlocks
+        Roxie heap memory.</para>
+
+        <para>Examples:</para>
+
+        <programlisting>ecl roxie memunlock mycluster
+</programlisting>
+      </sect2>
+
+      <sect2 id="CT-RCTL-GETMEMLOCK" role="brk">
+        <title>ecl roxie getmemlocked</title>
+
+        <para><emphasis role="bold">ecl roxie getmemlocked
+        &lt;process_cluster&gt;</emphasis></para>
+
+        <informaltable colsep="1" frame="all" rowsep="1">
+          <tgroup cols="2">
+            <colspec colwidth="80.50pt" />
+
+            <colspec />
+
+            <tbody>
+              <row>
+                <entry><emphasis>Options</emphasis></entry>
+
+                <entry></entry>
+              </row>
+
+              <row>
+                <entry><emphasis>process_cluster</emphasis></entry>
+
+                <entry>The Roxie process cluster to getmemlocked
+                status</entry>
+              </row>
+            </tbody>
+          </tgroup>
+        </informaltable>
+
+        <para>The <emphasis>ecl roxie getmemlocked</emphasis> command
+        retrieves the Roxie heap memory lock status. You can run the ECL
+        command-line on any host as long as it knows which ESP to connect to.
+        This is the ‘best’ way to check if Roxie memory is locked.</para>
+
+        <para>Examples:</para>
+
+        <programlisting>ecl roxie getmemlocked mycluster
+</programlisting>
+
+        <sect3>
+          <title>Manual Memory Lock</title>
+
+          <para>If running the latest point release of the HPCC Systems
+          platform, version 8.6 or later, and you want memory locked on Roxie
+          start up. Add the following to the <emphasis
+          role="strong">environment.xml</emphasis> in the RoxieCluster
+          section.</para>
+
+          <programlisting>    heapLockMemory="true"</programlisting>
+        </sect3>
```

Comment:
no mention of helm option ?

Reply:
Could not determine the helm/yaml value equivalent to set heapLockMemory=true thus removing entire section from this PR.